### ISSUE TITLE
QR code generation with logo overlay makes the code unreadable

### ISSUE THREAD

**Issue by saeedsryhini** [aboullaite/useful-scripts#2]
I tried the code inside QrCode-generation.java, it works correctly but the problem is that the generated QR code is unreadable by QR Scanner app.

I also tried reading the QR code you published in the article:
https://aboullaite.me/generate-qrcode-with-logo-image-using-zxing/
but that QR code was unreadable as well.

What is the use of such code if it makes the QR code unreadable?

**Comment by SDixit-MSC** [comment]
True. This is a total waste of time. it generates an image not a readable QR code.

**Comment by SimonScholz** [comment]
You might want to try this library:
https://github.com/SimonScholz/qr-code-with-logo

![image](https://github.com/aboullaite/useful-scripts/assets/7559962/183e16f3-2056-476b-9f8c-a060a6c4f892)

It also provides a user interface, which can generate QR code and also actual Java or Kotlin code to generate QR codes.

![image](https://github.com/aboullaite/useful-scripts/assets/7559962/384bb99b-9227-4c66-93de-dfe601da076f)
